c/build_all/opensuse/build.sh

```diff
@@ -0,0 +1,369 @@
+
+#!/bin/bash
```

Comment:
This whole file is essentially duplicated. What are the actual changes for OpenSUSE. Can they be extracted so that we do not duplicate the whole build folder?